src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -2938,6 +2932,16 @@ MulticopterPositionControl::generate_attitude_setpoint(float dt)
 	_att_sp.timestamp = hrt_absolute_time();
 }
 
+bool MulticopterPositionControl::manual_wants_takeoff()
+{
+	const bool has_manual_control_present = _control_mode.flag_control_manual_enabled && _manual.timestamp > 0;
+
+	// If the throttle stick is well above 50%, so above 62.5% (50% + 0.15 * 50%) we trigger manual takeoff.
+	const bool manual_wants_takeoff = (has_manual_control_present && _manual.z > 0.15f);
```

Comment:
manual.z goes from 0 to 1. We should put it to 0.65

Reply:
Please fix the comment as well!